src/servers/http_server.cc

```diff
@@ -1077,6 +1081,42 @@ HTTPAPIServer::InferResponseAlloc(
   return nullptr;  // Success
 }
 
+TRITONSERVER_Error*
```

Comment:
Add comment here and before InferResponseAlloc above.
"Make sure to keep InferResponseAlloc and OutputBufferQuery logic in sync"